controller-server/src/main/java/com/yahoo/vespa/hosted/controller/notification/Notification.java

```diff
@@ -0,0 +1,61 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.controller.notification;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author freva
+ */
+public class Notification {
```

Comment:
Since this is the core type used in this feature, consider adding javadoc, . E.g. explain who can receive a notification and its main purpose.